lib/python/rosie/browser/__init__.py

```diff
@@ -50,11 +50,7 @@
 
 """
 
-import ast
```

Comment:
You might want to modify the docstring (line 36) to match what is going on.

Reply:
Good spot @wxtim . Happy to apply that change, but not too sure what I should change. As I have very little understanding of Rose, I've limited to just what the IDE pointed. Should we delete the whole paragraph, i.e.

```
The values you enter will be cast by Python's ast.literal_eval, so:
foo=100
will be cast to an integer, but:
bar="100"
will be cast to a string.
```

in this pull request too? If so just let me know and I will add an extra commit for that :)

Reply:
propose 
```python
"""
The values in the rose.conf will be treated as string or int depending on
whether you quote them, so:
size_history=100
will be cast to an integer, but:
size_history="100"
will be cast to a string.
This may cause a type error if the value is the wrong type.
"""
```
I'd have created a PR on your branch but Octocat Wan Kenobi said:
https://github.com/404